src/styles/product.scss

```
@@ -0,0 +1,27 @@
+@import "./colors";
+
+.product-box {
+  width: 285px;
+  height: 446px;
+  display: flex;
+  flex-direction: column;
+  background-color: $product-background;
+  text-align: left;
+  padding: 10px;
+
+  & .__image {
```

You dont need the space and dot
```suggestion
  & .__image {
```

Take in consideration that the resulting class on the html has to be:
```html
  className="product-box__image"
```

---

Wow sorry, i got it now...